flask_calendar/actions.py

```diff
@@ -5,13 +5,13 @@
 
 from flask import render_template, request, jsonify, redirect, abort, make_response
 
-import config
-from constants import SESSION_ID
-from gregorian_calendar import GregorianCalendar
-from calendar_data import CalendarData
-from authentication import Authentication
-from app_utils import (previous_month_link, next_month_link, new_session_id, add_session, authenticated,
-                       get_session_username, authorized)
+from .settings import config
```

Comment:
it is a best practice (and I really prefer it) to use absolute paths, and we don't need to specify `flask_calendar.xxx` on them, so could leave `from settings ...` etc.

It might sound biased but I'm quite used to absolute paths and it really helps when things grow in size to not worry about relative paths.

Reply:
No problem, let's avoid relative imports! I really appreciate the explanation :relaxed: